specification/hdinsight/resource-manager/readme.md

```diff
@@ -36,13 +36,34 @@ tag: package-2018-06-preview
  ``` yaml
  directive:
    - suppress: DefinitionsPropertiesNamesCamelCase
-     reason: This would require a breaking change, and the capabilities API was removed in version 2018-06-01-preview.
-     from: Microsoft.HDInsight/preview/2015-03-01-1/preview/locations.json
+     reason: This would require a breaking change, and need to be consistent with the response from RP side.
+     from: Microsoft.HDInsight/preview/2015-03-01-preview/locations.json
      where:
        - $.definitions.CapabilitiesResult.properties.vmSize_filters
        - $.definitions.RegionalQuotaCapability.properties.cores_available
        - $.definitions.RegionalQuotaCapability.properties.cores_used
        - $.definitions.RegionalQuotaCapability.properties.region_name
+       - $.definitions.QuotaCapability.properties.cores_used
+       - $.definitions.QuotaCapability.properties.max_cores_allowed
+       - $.definitions.VmSizeCompatibilityFilter.properties.ClusterVersions
```

Comment:
What's the problem with `$.definitions.VmSizeCompatibilityFilter.properties.ClusterVersions` and all the other ` $.definitions.VmSizeCompatibilityFilter.properties.*` property names. The casing looks fine - do they need to be added to this suppression?

Reply:
> What's the problem with `$.definitions.VmSizeCompatibilityFilter.properties.ClusterVersions` and all the other ` $.definitions.VmSizeCompatibilityFilter.properties.*` property names. The casing looks fine - do they need to be added to this suppression?

@antmarti-microsoft In fact swagger violation tools require lowerCamelCase. So the recommended usage is clusterVersions etc. By the way, this PR don't modify $.definitions.VmSizeCompatibilityFilter.properties.* property of readme.md file. They have been in master branch.